content/Hardware Support/Giga/Update-Wi-Fi-firmware-on-GIGA-R1-WiFi.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Update Wi-Fi firmware on GIGA R1 WiFi"
+---
+
+In this article, we will go through a step-by-step process on how to get the latest Wi-Fi firmware on your GIGA R1 WiFi board.
+
+## Before you begin
+
+* Download the **Arduino Mbed OS Giga Boards** using the [Boards Manager in Arduino IDE](https://support.arduino.cc/hc/en-us/articles/360016119519-Add-boards-to-Arduino-IDE).
```

Comment:
The word "Download" might be confusing since the "Add boards to Arduino IDE" article uses the word "Add" instead.

```suggestion
*Add the **Arduino Mbed OS Giga Boards** to Arduino IDE using the [Boards Manager](https://support.arduino.cc/hc/en-us/articles/360016119519-Add-boards-to-Arduino-IDE).
```